factories/HeroImage.php

```diff
@@ -26,7 +26,7 @@ public function create($limit = 1, $flatten = false, $options = [])
             $data[$i] = [
                 'relative_url' => '/styleguide/image/1600x580?text=1600x580%20('.$i.')',
                 'title' => $this->faker->sentence,
-                'description' => '<p>' . $this->faker->text(100) . ' <a href="#">'. $this->faker->sentence(3)  .'</a></p>',
+                'description' => '<p>' . $this->faker->text(100) . ' <a href="https://wayne.edu">'. $this->faker->sentence(3)  .'</a></p>',
```

Comment:
Perhaps we should have all wayne.edu URLs with or without the trailing slash for consistency?